sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ClassName.cs

```diff
@@ -68,7 +68,7 @@ protected sealed override void Initialize(SonarAnalysisContext context)
                     }
 
                     string suggestion;
-                    if (TryGetChangedName(identifier.ValueText, typeDeclaration, c.SemanticModel.Compilation.IsTest(), out suggestion))
+                    if (TryGetChangedName(identifier.ValueText, typeDeclaration, c.IsTest(), out suggestion))

```

Comment:
You should delete the old `IsTest` on Compilation if it is not used anymore. (disregard if you already did it)